### DISCUSSION TITLE
The user 'System:PublicAccess;aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa' is not authorized to access this resource

### DISCUSSION THREAD

**Discussion by karenpayneoregon** [community/community#25300] [Actions]
<p>This is my 2nd post, first one got hidden from one of your bots over ten hours ago and is still not visible.</p>
<p><strong>Problem statement</strong></p>
<ol>
<li>Made a change to index.html on a <code>io</code> page.</li>
<li>Commit, Pushed</li>
<li>Got the following on build/deploy</li>
</ol>
<pre><code class="lang-auto">{
"$id": "1",
"innerException": null,
"message": "The user 'System:PublicAccess;aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa' is not authorized to access this resource.",
"typeName": "Microsoft.TeamFoundation.Framework.Server.UnauthorizedRequestException, Microsoft.TeamFoundation.Framework.Server",
"typeKey": "UnauthorizedRequestException",
"errorCode": 0,
"eventId": 3000
}
</code></pre>
<p>I have a current SSH key and GPG created yesterday. Deploy key is the SSH key.</p>
<p>I have no clue why this is happening and how to resolve this issue.</p>

**Comment by jfullstackdev** [comment]
<p>your last successful build was this,</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181090146-9f406a7b-f211-4b6e-8011-cee69d0fafdb.png" alt="image" data-base62-sha1="fqSQXybGO3rOA61n6QwO5wbeBEq" width="425" height="84"></p>
<p>think of all those changes that you’ve made and trace the problem</p>

**Comment by karenpayneoregon** [comment]
<p>I tried to revert back prior to 10 days, zero luck so I did a backup of the repo, removed 99 percent of the code and it deployed.</p>
<p>The repo is something I do from time to time in VS Code. So I’m going to put things back one section at a time to see if I can figure out the why.</p>
<p>In the future is there steps I can follow to learn why I’ve gotten this issue?</p>

**Comment by jfullstackdev** [comment]
<p>yup, that’s actually the technique, small changes and make sure it’s working, when problem arises, you know where to find it,</p>

**Comment by karenpayneoregon** [comment, marked as answer]
<p>Finally had time to sit down and work with this. I removed pretty much all the files, added them back in one by one newer files first. One specific file failed so I piece meal adding  sections back in. At about 80 percent down VS Code placed a right triangle on a blank line, removed the blank line, triangle moved to the next line. So I took that section and placed it into a new file, linked back to the first page.</p>
<p>At this point the build/deploy worked.</p>
<p>Added all other files back in, build/deploy went active.</p>
<p>So the one thing I don’t know about, why the <strong>right triangle pointing to an issue</strong> but hovering over the right triangle <strong>gave no indication of what was wrong</strong>.</p>

**Reply by galenseilis** [reply]
I am getting the same error right now. And I am lamenting that there is not a loud failure on what is not correct and where.

**Comment by Primetalk** [comment]
I've got the same message at deploy stage.

> The user 'System:PublicAccess;aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa' is not authorized to access this resource.

The message seems to be misleading. It would be great if the message contained some context information of what went wrong.
